packages/aws-client/package.json

```diff
@@ -44,7 +44,7 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@aws-sdk/client-api-gateway": "^3.447.0",
+    "@aws-sdk/client-api-gateway": "^3.499.0",
```

Comment:
Was this a vulnerability?

Reply:
3.447 imported crypto-js at a lower version as a dependency (whichis the actual vulnerability)
. 3.499 doens't

Reply:
Ah ok, got it.